src/betterproto/plugin/compiler.py

```diff
@@ -31,7 +32,20 @@ def outputfile_compiler(output_file: OutputTemplate) -> str:
     )
     template = env.get_template("template.py.j2")
 
+    code = template.render(output_file=output_file)
+    code = isort.api.sort_code_string(
+        code=code,
+        show_diff=False,
+        py_version=36,
+        profile="black",
+        force_single_line=True,
```

Comment:
This needs turning off to match the settings in pyproject.toml

Reply:
This is done.